doc/man7/life_cycle-digest.pod

```diff
@@ -0,0 +1,221 @@
+=pod
+
+=head1 NAME
+
+life_cycle-digest - The digest algorithm life-cycle
+
+=head1 DESCRIPTION
+
+All message digests (MDs) go through a number of stages in their life-cycle:
+
+=over 4
+
+=item start
+
+This state represents the MD before it has been allocated.  It is the
+starting state for any life-cycle transitions.
+
+=item newed
```

Comment:
created?

Reply:
Using the same words as the rest....
Created would be viable nonetheless.

Reply:
It is just the transition back to a 'newed' state seems a bit funny (dont care too much either way)

Reply:
I've used _newed_ throughout all the documentation since it is the result of a XXX_new() call.

I'd prefer to leave it as it is :grinning: